README.md

```diff
@@ -20,7 +20,7 @@ authorized users can then invoke these quantum programs by simply passing in the
 :rocket: Qiskit Runtime is now available on all IBM Quantum systems. Open-access users
 can only execute runtime programs on `ibmq_qasm_simulator`, but premium users can execute them on
 any systems to which they have access. You are an open-access user if `ibm-q/open/main` is your 
-_only_ provider.
+_only_ provider and you are a premium user if you have access to more than one provider.
```

Comment:
My previous comment was about explaining what a premium user is before or in the same paragraph it's first used :). 

But after reading the first part of the README, I now propose we keep the "authorized users" wording like before (both in README and tutorial), and change only the section where authorization is specifically mentions. For example, line 13 will continue to say `Qiskit Runtime allows authorized users ... `, and here we can say 

```
Qiskit Runtime is now available on all IBM Quantum systems. Open-access users
can only execute runtime programs on `ibmq_qasm_simulator`, but premium users can execute them on
any systems to which they have access. Premium users are also allowed to upload Qiskit Runtime programs. 
You are an open-access user if `ibm-q/open/main` is your _only_ provider, and you are a premium user if you have access to more than one provider.
```

This way users only need to look at the authorization section for any changes, and we only need to update that section for future changes.